snmp/datadog_checks/snmp/data/default_profiles/tripplite.yaml

```diff
@@ -0,0 +1,8 @@
+extends:
```

Comment:
note: `tripplite.yaml` is also going to be used in tripplite-ups: https://github.com/DataDog/integrations-core/pull/15397